libs/hdf-converters/src/converters-from-hdf/asff/transformers.ts

```diff
@@ -487,6 +512,16 @@ function createProfileInfoFindingFields(
       }
     });
   });
+  const charLimit = 30000;
+  const passThroughObj = _.get(hdf, 'passthrough');
+  if (passThroughObj instanceof Object) {
+    pushSplitString(
+      escapeForwardSlashes(JSON.stringify(passThroughObj)),
+      charLimit,
+      typesArr,
+      'Execution/passthrough'
```

Comment:
Add handling for converting this back into hdf in `src/asff-mapper/case-hdf2asff.ts`